robot_skills/src/robot_skills/arms.py

```diff
@@ -175,6 +219,34 @@ def __init__(self, robot_name, tf_listener, side):
             "/" + robot_name + "/" + self.side + "_arm/grasp_target",
             visualization_msgs.msg.Marker, queue_size=10)
 
+    def has_gripper_type(self, gripper_type):
+        """
+        Query the arm for having the proper gripper type.
+
+        :param gripper_type: Should be constant 'yes' (we have one). At some point in the
+                             future it may be useful to have a proper type.
+        :return: Whether the arm has the requested type of gripper.
+        """
+        return gripper_type == 'yes'
```

Comment:
Maybe add an Enum with all available gripper types. Eg. `pinch` or `parallel` or `suction`

Reply:
Added various `GRIPPER_TYPE_...` constants in `arms.py`